getting-started.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Getting started"
+description: Get started with Coder.
+icon: '<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" viewBox="0 0 16 16" width="16" xml:space="preserve"><path d="M1 2a1 1 0 0 1 1-1h11a1 1 0 0 1 1 1v1h.5A1.5 1.5 0 0 1 16 4.5v7a1.5 1.5 0 0 1-1.5 1.5h-.55a2.5 2.5 0 0 1-2.45 2h-8A2.5 2.5 0 0 1 1 12.5V2zm13 10h.5a.5.5 0 0 0 .5-.5v-7a.5.5 0 0 0-.5-.5H14v8z"/></svg>'
+---
+
+We’re excited for you to be part of the growing community of Coder users, and we
+wanted to provide an onboarding guide to ensure you have a great experience.
+Coder’s [documentation is completely open-source](https://github.com/cdr/docs),
+so please feel free to contribute and provide feedback as you desire.
+
+## Setup and deploy Coder
+
+To get started with Coder, you’ll need to [deploy Coder](./setup/index.md) to a
+Kubernetes cluster. We have documentation to help you [create a
+cluster](./setup/kubernetes/index.md) if needed. Once you have a cluster, you
+can [install Coder via Helm](./setup/installation.md).
+
+## Configure Coder
+
+After you deploy Coder, you’ll need to [upload your license
+file](./setup/configuration.md) before you can configure the application (you
+can get a trial license for free [here](https://coder.com/trial)). Once logged
+in, you’ll be able to access the [administration management](.//admin/index.md)
+menu to set up things such as [access
+controls](.//admin/access-control/index.md) with OpenID Connect (OIDC),
```

Comment:
the `//` doesn't seem correct here? same for another instance above

Reply:
```suggestion
controls](./admin/access-control/index.md) with OpenID Connect (OIDC),
```